rebuild.sh

```diff
@@ -5,6 +5,9 @@ wget http://www.cpan.org/src/5.0/perl-5.26.2.tar.gz
 tar xzvf perl-5.26.2.tar.gz
 cd perl-5.26.2
 
+# Apply patch for fixing Time::Local tests entering year 2020  (CPAN RT#124787)
+curl -fsSL https://rt.cpan.org/Public/Ticket/Attachment/1776857/956088/0001-Fix-Time-Local-tests.patch | patch -p1
```

Comment:
This card is so large already, why don't we just store a copy of the patch itself?

Reply:
Yep thinking about that as well, probably better to keep it and maybe later patches in a directory to `cat | patch` through...

Reply:
Now added as a local copy to `zcat | patch || exit 1` with.